pennylane_lightning/core/src/simulators/lightning_gpu/gates/tests/Test_StateVectorCudaManaged_Param.cpp

```diff
@@ -35,6 +36,7 @@
 namespace {
 using namespace Pennylane;
 using namespace Pennylane::LightningGPU;
+std::mt19937_64 re{1337};
```

Comment:
Do we want this defined in the translation unit, or local to the tests it is used in?

Reply:
Yes, that exactly. I know this has caused problems in the Python layer in the past.

Reply:
Ok, I'll fix this except for GPU-MPI which I'm not testing in this PR.